wordle-web/src/components/__tests__/HelloWorld.spec.ts

```diff
@@ -0,0 +1,11 @@
+import { describe, it, expect } from 'vitest'
+
+import { mount } from '@vue/test-utils'
+import HelloWorld from '../HelloWorld.vue'
+
+describe('HelloWorld', () => {
```

Comment:
Seems to appropriately run on the pipeline and pass each time. Though it seems kind of redundant since the `HelloWorld.vue` component isn't used. You could easily add the letter object/tests if you wanted to, since they gave us the code for it. Or write tests of your own similar to this one, testing that for example your cards have the right filler text or something.